themes-default/slim/src/components/helpers/config-toggle-slider.vue

```diff
@@ -63,7 +63,7 @@ export default {
     methods: {
         updateValue() {
             const { localChecked } = this;
-            this.$emit('input', localChecked);
+            this.$emit('input', localChecked, this);
```

Comment:
this looks a bit strange to me, why is it needed?

Reply:
It's used here: https://github.com/pymedusa/Medusa/pull/7890/files?file-filters%5B%5D=.vue#diff-de8df93ed1c5bebdca47795229ddcfb0R88

And here:
https://github.com/pymedusa/Medusa/pull/7890/files?file-filters%5B%5D=.vue#diff-de8df93ed1c5bebdca47795229ddcfb0R449

`this` is a reference to the element itself. I want to know which "config-toggle-slider" component was enabled. So I can disable the other two.

Reply:
I changed it, so it uses a manual paramater now.